### PR TITLE
Windows compatibility

### PR DESCRIPTION
**Pull request by ErikBjare**

### CODE REVIEW

File: aw_qt/manager.py [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 import logging
 import subprocess
-from subprocess import PIPE
+from subprocess import PIPE, STDOUT
 from typing import Optional
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger("aw.qt.manager")
 
+# TODO: Proper detection, NEEDS TO BE FIXED BEFORE MERGE WITH MASTER
```
Comment (ErikBjare) [inline aw_qt/manager.py]: - [ ] This must not be merged into master without fix

**Comment by johan-bjareholt** [comment]
@ErikBjare What's up with this old branch?

**Comment by ErikBjare** [comment]
@johan-bjareholt I'm probably going to close this PR without merge soon.

**Comment by ErikBjare** [comment]
Windows compatibility landed in https://github.com/ActivityWatch/aw-qt/pull/17

**Pull request closed without merging**